### PR TITLE
Reverse String Program

### PR DESCRIPTION
**Pull request by tejashrighogare**

### CODE REVIEW

File: MediumDifficultyProgram6/RevString/RevString.java [hunk 1]
```diff
@@ -0,0 +1,27 @@
+package RevString;
+
+public class RevString {
+    /*
+    Q6. Write a program to reverse the string without changing the position of special character.
+ */
+    public static void main(String[] args) {
+        String result = Reverse("Ro@hitshetty@");
```
Comment (prashantkawa) [inline MediumDifficultyProgram6/RevString/RevString.java]: Please check the result for this input, char's are getting replaced with special character

**Review by prashantkawa — CHANGES_REQUESTED** [review]

File: MediumDifficultyProgram6/RevString/RevString.java [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+        System.out.println(result);
+    }
+
+    static String Reverse(String inputString) {
+        String result = "";
+        String specialCharatcterList = "@,#,$,<,>,?,/,^,&,*,(,),{,},[,],:,.";
```
Comment (rohitlamsonge66) [inline MediumDifficultyProgram6/RevString/RevString.java]: Do not use this as static. write the code for this.

File: MediumDifficultyProgram6/RevString/RevString.java (hunk 1 above)
Comment (tejashrighogare) [reply MediumDifficultyProgram6/RevString/RevString.java]: Yes, Changed code as per the comment

File: MediumDifficultyProgram6/RevString/RevString.java (hunk 2 above)
Comment (tejashrighogare) [reply MediumDifficultyProgram6/RevString/RevString.java]: replaced static values for special characters and used code

File: MediumDifficultyProgram6/RevString/readMe [hunk 3]
```diff
@@ -0,0 +1,4 @@
+Input = Ro@hitshetty
```
Comment (prashantkawa) [inline MediumDifficultyProgram6/RevString/readMe:1]: Try with different input's  Ex. "Ro@hitshetty@"

**Review by prashantkawa — CHANGES_REQUESTED** [review]

**Review by rohitlamsonge66 — APPROVED** [review]

**Review by prashantkawa — APPROVED** [review]
Approving PR

**Pull request merged**